src/main/java/org/mockito/internal/runners/StrictRunner.java

```diff
@@ -16,7 +16,7 @@ public class StrictRunner implements InternalRunner {
 
     private final Class<?> testClass;
     private final InternalRunner runner;
-    private boolean filterRequested;
+    private volatile boolean testsSkipped;
```

Comment:
Why does this need to be `volatile`? We try to stay away of that as much as we can, as multi-threading is already a difficult story with Mockito. As I see it, there is no need for this to be shared across instances, so I think this is not necessary?

Reply:
Ah thanks, good catch! I wasn't familiar with the threading model at play here and wanted to be conservative. I've removed the `volatile` modifier.